code/modules/mob/living/living_defense.dm

```diff
@@ -437,3 +437,40 @@
 
 /mob/living/proc/force_hit_projectile(obj/projectile/projectile)
 	return FALSE
+
+/mob/living/proc/get_weapon_inaccuracy_modifier(atom/target, obj/item/gun/weapon)
+	. = 0
+	if(HAS_TRAIT(src, TRAIT_POOR_AIM)) //nice shootin' tex
+		. += 25
+	// Unwielded weapons
+	if(!weapon.is_wielded && weapon.requires_wielding)
+		. += weapon.spread_unwielded
+	// Nothing to hold onto, slight penalty for flying around in space
+	var/default_speed = get_config_multiplicative_speed() + CONFIG_GET(number/movedelay/run_delay)
```

Comment:
get_config_multiplicative_speed() doesn't take into account the run delay, so we have to add it on here.